diff -up gdal-3.3.0/swig/python/GNUmakefile.4~ gdal-3.3.0/swig/python/GNUmakefile
--- gdal-3.3.0/swig/python/GNUmakefile.4~	2021-05-04 18:51:30.539323948 +0200
+++ gdal-3.3.0/swig/python/GNUmakefile	2021-05-04 18:58:00.697393776 +0200
@@ -96,7 +96,7 @@ generate: veryclean
 build: .generated_files_up_to_date
 	rm -f setup_vars.ini
 	echo 'GNM_ENABLED=$(GNM_ENABLED)' >> setup_vars.ini
-	$(PYTHON) setup.py build
+	CC=clang CXX=clang++ LDFLAGS=-lpython3.11 $(PYTHON) setup.py build
 
 egg:
 	$(PYTHON) setup.py bdist_egg
