--- gdal-2.3.2/GDALmake.opt.in.orig	2018-11-19 21:44:02.778592425 +0000
+++ gdal-2.3.2/GDALmake.opt.in	2018-11-19 21:44:10.993591900 +0000
@@ -12,7 +12,7 @@
 ifeq ($(HAVE_LIBTOOL),yes)
 LIBTOOL_COMPILE_CC =	$(LIBTOOL) --mode=compile --silent --tag=CC
 LIBTOOL_COMPILE_CXX =	$(LIBTOOL) --mode=compile --silent --tag=CXX
-LIBTOOL_LINK	=	$(LIBTOOL) --mode=link --silent
+LIBTOOL_LINK	=	$(LIBTOOL) --mode=link --silent --tag=CXX
 LIBTOOL_INSTALL	=	$(LIBTOOL) --mode=install --silent
 LIBTOOL_FINISH	=	$(LIBTOOL) --mode=finish --silent
 LIBTOOL_CLEAN	=	$(LIBTOOL) --mode=clean --silent
