diff -up gdal-3.4.1/swig/python/GNUmakefile.3~ gdal-3.4.1/swig/python/GNUmakefile
--- gdal-3.4.1/swig/python/GNUmakefile.3~	2022-01-04 14:49:40.874264928 +0100
+++ gdal-3.4.1/swig/python/GNUmakefile	2022-01-04 14:49:58.502297758 +0100
@@ -104,7 +104,7 @@ egg:
 
 site_package_dir=$(shell $(PYTHON) -c "from trimmedsysconfig import get_python_lib;print(get_python_lib(prefix=\"$(PREFIX)\"))")
 
-ifeq ($(PY_HAVE_SETUPTOOLS),1)
+ifeq ($(PY_HAVE_SETUPTOOLS),0)
     setup_opts=--single-version-externally-managed --record=record.txt
 endif
 
