--- gdal-2.2.1/swig/python/GNUmakefile.0005~	2017-08-29 14:31:29.146982766 +0200
+++ gdal-2.2.1/swig/python/GNUmakefile	2017-08-29 14:32:56.836827903 +0200
@@ -65,7 +65,7 @@ generate: ${WRAPPERS} gdal_array_wrap.cp
 build: extensions/gdal_wrap.cpp
 	rm -f setup_vars.ini
 	echo 'GNM_ENABLED=$(GNM_ENABLED)' >> setup_vars.ini
-	$(PYTHON) setup.py build
+	CC=clang CXX=clang++ LDFLAGS="-lpython3.8" $(PYTHON) setup.py build
 
 egg:
 	$(PYTHON) setup.py bdist_egg 
