--- gdal-2.2.1/GDALmake.opt.in.libtoolsux~	2017-08-29 13:58:12.951503574 +0200
+++ gdal-2.2.1/GDALmake.opt.in	2017-08-29 14:00:44.231071347 +0200
@@ -12,7 +12,7 @@ LIBTOOL	=	@LIBTOOL@
 ifeq ($(HAVE_LIBTOOL),yes)
 LIBTOOL_COMPILE_CC =	$(LIBTOOL) --mode=compile --tag=CC
 LIBTOOL_COMPILE_CXX =	$(LIBTOOL) --mode=compile --tag=CXX
-LIBTOOL_LINK	=	$(LIBTOOL) --mode=link
+LIBTOOL_LINK	=	$(LIBTOOL) --mode=link --tag=CXX
 LIBTOOL_INSTALL	=	$(LIBTOOL) --mode=install
 LIBTOOL_FINISH	=	$(LIBTOOL) --mode=finish --silent
 LIBTOOL_CLEAN	=	$(LIBTOOL) --mode=clean
