diff -rupN gdal-3.11.3.old/swig/python/gdal-utils/osgeo_utils/auxiliary/array_util.py gdal-3.11.3/swig/python/gdal-utils/osgeo_utils/auxiliary/array_util.py
--- gdal-3.11.3.old/swig/python/gdal-utils/osgeo_utils/auxiliary/array_util.py	2025-07-12 12:33:49.000000000 +0200
+++ gdal-3.11.3/swig/python/gdal-utils/osgeo_utils/auxiliary/array_util.py	2025-08-21 17:52:17.238879749 +0200
@@ -1,4 +1,4 @@
-#!/usr/bin/env python3
+#!/usr/bin/python3
 # -*- coding: utf-8 -*-
 # ******************************************************************************
 #
diff -rupN gdal-3.11.3.old/swig/python/gdal-utils/osgeo_utils/auxiliary/base.py gdal-3.11.3/swig/python/gdal-utils/osgeo_utils/auxiliary/base.py
--- gdal-3.11.3.old/swig/python/gdal-utils/osgeo_utils/auxiliary/base.py	2025-07-12 12:33:49.000000000 +0200
+++ gdal-3.11.3/swig/python/gdal-utils/osgeo_utils/auxiliary/base.py	2025-08-21 17:52:25.859305996 +0200
@@ -1,4 +1,4 @@
-#!/usr/bin/env python3
+#!/usr/bin/python3
 # -*- coding: utf-8 -*-
 # ******************************************************************************
 #
diff -rupN gdal-3.11.3.old/swig/python/gdal-utils/osgeo_utils/auxiliary/batch_creator.py gdal-3.11.3/swig/python/gdal-utils/osgeo_utils/auxiliary/batch_creator.py
--- gdal-3.11.3.old/swig/python/gdal-utils/osgeo_utils/auxiliary/batch_creator.py	2025-07-12 12:33:49.000000000 +0200
+++ gdal-3.11.3/swig/python/gdal-utils/osgeo_utils/auxiliary/batch_creator.py	2025-08-21 17:52:33.525762132 +0200
@@ -1,4 +1,4 @@
-#!/usr/bin/env python3
+#!/usr/bin/python3
 # -*- coding: utf-8 -*-
 # ******************************************************************************
 #
diff -rupN gdal-3.11.3.old/swig/python/gdal-utils/osgeo_utils/auxiliary/color_palette.py gdal-3.11.3/swig/python/gdal-utils/osgeo_utils/auxiliary/color_palette.py
--- gdal-3.11.3.old/swig/python/gdal-utils/osgeo_utils/auxiliary/color_palette.py	2025-07-12 12:33:49.000000000 +0200
+++ gdal-3.11.3/swig/python/gdal-utils/osgeo_utils/auxiliary/color_palette.py	2025-08-21 17:52:40.481569425 +0200
@@ -1,4 +1,4 @@
-#!/usr/bin/env python3
+#!/usr/bin/python3
 # -*- coding: utf-8 -*-
 # ******************************************************************************
 #
diff -rupN gdal-3.11.3.old/swig/python/gdal-utils/osgeo_utils/auxiliary/color_table.py gdal-3.11.3/swig/python/gdal-utils/osgeo_utils/auxiliary/color_table.py
--- gdal-3.11.3.old/swig/python/gdal-utils/osgeo_utils/auxiliary/color_table.py	2025-07-12 12:33:49.000000000 +0200
+++ gdal-3.11.3/swig/python/gdal-utils/osgeo_utils/auxiliary/color_table.py	2025-08-21 17:52:53.589683437 +0200
@@ -1,4 +1,4 @@
-#!/usr/bin/env python3
+#!/usr/bin/python3
 # -*- coding: utf-8 -*-
 # ******************************************************************************
 #
diff -rupN gdal-3.11.3.old/swig/python/gdal-utils/osgeo_utils/auxiliary/extent_util.py gdal-3.11.3/swig/python/gdal-utils/osgeo_utils/auxiliary/extent_util.py
--- gdal-3.11.3.old/swig/python/gdal-utils/osgeo_utils/auxiliary/extent_util.py	2025-07-12 12:33:49.000000000 +0200
+++ gdal-3.11.3/swig/python/gdal-utils/osgeo_utils/auxiliary/extent_util.py	2025-08-21 17:53:01.466753809 +0200
@@ -1,4 +1,4 @@
-#!/usr/bin/env python3
+#!/usr/bin/python3
 # -*- coding: utf-8 -*-
 # ******************************************************************************
 #
diff -rupN gdal-3.11.3.old/swig/python/gdal-utils/osgeo_utils/auxiliary/gdal_argparse.py gdal-3.11.3/swig/python/gdal-utils/osgeo_utils/auxiliary/gdal_argparse.py
--- gdal-3.11.3.old/swig/python/gdal-utils/osgeo_utils/auxiliary/gdal_argparse.py	2025-07-12 12:33:49.000000000 +0200
+++ gdal-3.11.3/swig/python/gdal-utils/osgeo_utils/auxiliary/gdal_argparse.py	2025-08-21 17:53:07.500840293 +0200
@@ -1,4 +1,4 @@
-#!/usr/bin/env python3
+#!/usr/bin/python3
 # -*- coding: utf-8 -*-
 # ******************************************************************************
 #
diff -rupN gdal-3.11.3.old/swig/python/gdal-utils/osgeo_utils/auxiliary/numpy_util.py gdal-3.11.3/swig/python/gdal-utils/osgeo_utils/auxiliary/numpy_util.py
--- gdal-3.11.3.old/swig/python/gdal-utils/osgeo_utils/auxiliary/numpy_util.py	2025-07-12 12:33:49.000000000 +0200
+++ gdal-3.11.3/swig/python/gdal-utils/osgeo_utils/auxiliary/numpy_util.py	2025-08-21 17:53:14.267081529 +0200
@@ -1,4 +1,4 @@
-#!/usr/bin/env python3
+#!/usr/bin/python3
 # -*- coding: utf-8 -*-
 # ******************************************************************************
 #
diff -rupN gdal-3.11.3.old/swig/python/gdal-utils/osgeo_utils/auxiliary/osr_util.py gdal-3.11.3/swig/python/gdal-utils/osgeo_utils/auxiliary/osr_util.py
--- gdal-3.11.3.old/swig/python/gdal-utils/osgeo_utils/auxiliary/osr_util.py	2025-07-12 12:33:49.000000000 +0200
+++ gdal-3.11.3/swig/python/gdal-utils/osgeo_utils/auxiliary/osr_util.py	2025-08-21 17:53:21.717314038 +0200
@@ -1,4 +1,4 @@
-# !/usr/bin/env python3
+# !/usr/bin/python3
 ###############################################################################
 #
 #  Project:  GDAL utils.auxiliary
diff -rupN gdal-3.11.3.old/swig/python/gdal-utils/osgeo_utils/auxiliary/progress.py gdal-3.11.3/swig/python/gdal-utils/osgeo_utils/auxiliary/progress.py
--- gdal-3.11.3.old/swig/python/gdal-utils/osgeo_utils/auxiliary/progress.py	2025-07-12 12:33:49.000000000 +0200
+++ gdal-3.11.3/swig/python/gdal-utils/osgeo_utils/auxiliary/progress.py	2025-08-21 17:53:46.463845695 +0200
@@ -1,4 +1,4 @@
-#!/usr/bin/env python3
+#!/usr/bin/python3
 ###############################################################################
 #
 # Project:  GDAL
diff -rupN gdal-3.11.3.old/swig/python/gdal-utils/osgeo_utils/auxiliary/raster_creation.py gdal-3.11.3/swig/python/gdal-utils/osgeo_utils/auxiliary/raster_creation.py
--- gdal-3.11.3.old/swig/python/gdal-utils/osgeo_utils/auxiliary/raster_creation.py	2025-07-12 12:33:49.000000000 +0200
+++ gdal-3.11.3/swig/python/gdal-utils/osgeo_utils/auxiliary/raster_creation.py	2025-08-21 17:53:53.960237641 +0200
@@ -1,4 +1,4 @@
-#!/usr/bin/env python3
+#!/usr/bin/python3
 # -*- coding: utf-8 -*-
 # ******************************************************************************
 #
diff -rupN gdal-3.11.3.old/swig/python/gdal-utils/osgeo_utils/auxiliary/rectangle.py gdal-3.11.3/swig/python/gdal-utils/osgeo_utils/auxiliary/rectangle.py
--- gdal-3.11.3.old/swig/python/gdal-utils/osgeo_utils/auxiliary/rectangle.py	2025-07-12 12:33:49.000000000 +0200
+++ gdal-3.11.3/swig/python/gdal-utils/osgeo_utils/auxiliary/rectangle.py	2025-08-21 17:54:01.473464175 +0200
@@ -1,4 +1,4 @@
-#!/usr/bin/env python3
+#!/usr/bin/python3
 # -*- coding: utf-8 -*-
 # ******************************************************************************
 #
diff -rupN gdal-3.11.3.old/swig/python/gdal-utils/osgeo_utils/auxiliary/util.py gdal-3.11.3/swig/python/gdal-utils/osgeo_utils/auxiliary/util.py
--- gdal-3.11.3.old/swig/python/gdal-utils/osgeo_utils/auxiliary/util.py	2025-07-12 12:33:49.000000000 +0200
+++ gdal-3.11.3/swig/python/gdal-utils/osgeo_utils/auxiliary/util.py	2025-08-21 17:52:03.436861018 +0200
@@ -1,4 +1,4 @@
-#!/usr/bin/env python3
+#!/usr/bin/python3
 # -*- coding: utf-8 -*-
 # ******************************************************************************
 #
diff -rupN gdal-3.11.3.old/swig/python/gdal-utils/osgeo_utils/gdal2tiles.py gdal-3.11.3/swig/python/gdal-utils/osgeo_utils/gdal2tiles.py
--- gdal-3.11.3.old/swig/python/gdal-utils/osgeo_utils/gdal2tiles.py	2025-07-12 12:33:50.000000000 +0200
+++ gdal-3.11.3/swig/python/gdal-utils/osgeo_utils/gdal2tiles.py	2025-08-21 17:50:40.295762611 +0200
@@ -1,4 +1,4 @@
-#!/usr/bin/env python3
+#!/usr/bin/python3
 # -*- coding: utf-8 -*-
 # ******************************************************************************
 #
diff -rupN gdal-3.11.3.old/swig/python/gdal-utils/osgeo_utils/gdal2xyz.py gdal-3.11.3/swig/python/gdal-utils/osgeo_utils/gdal2xyz.py
--- gdal-3.11.3.old/swig/python/gdal-utils/osgeo_utils/gdal2xyz.py	2025-07-12 12:33:49.000000000 +0200
+++ gdal-3.11.3/swig/python/gdal-utils/osgeo_utils/gdal2xyz.py	2025-08-21 17:50:47.727021813 +0200
@@ -1,4 +1,4 @@
-#!/usr/bin/env python3
+#!/usr/bin/python3
 ###############################################################################
 #
 # Project:  GDAL
diff -rupN gdal-3.11.3.old/swig/python/gdal-utils/osgeo_utils/gdalattachpct.py gdal-3.11.3/swig/python/gdal-utils/osgeo_utils/gdalattachpct.py
--- gdal-3.11.3.old/swig/python/gdal-utils/osgeo_utils/gdalattachpct.py	2025-07-12 12:33:49.000000000 +0200
+++ gdal-3.11.3/swig/python/gdal-utils/osgeo_utils/gdalattachpct.py	2025-08-21 17:50:55.202091861 +0200
@@ -1,4 +1,4 @@
-#!/usr/bin/env python3
+#!/usr/bin/python3
 # -*- coding: utf-8 -*-
 # ******************************************************************************
 #
diff -rupN gdal-3.11.3.old/swig/python/gdal-utils/osgeo_utils/gdal_calc.py gdal-3.11.3/swig/python/gdal-utils/osgeo_utils/gdal_calc.py
--- gdal-3.11.3.old/swig/python/gdal-utils/osgeo_utils/gdal_calc.py	2025-07-12 12:33:49.000000000 +0200
+++ gdal-3.11.3/swig/python/gdal-utils/osgeo_utils/gdal_calc.py	2025-08-21 17:49:20.265954772 +0200
@@ -1,4 +1,4 @@
-#!/usr/bin/env python3
+#!/usr/bin/python3
 # -*- coding: utf-8 -*-
 # ******************************************************************************
 #
diff -rupN gdal-3.11.3.old/swig/python/gdal-utils/osgeo_utils/gdalcompare.py gdal-3.11.3/swig/python/gdal-utils/osgeo_utils/gdalcompare.py
--- gdal-3.11.3.old/swig/python/gdal-utils/osgeo_utils/gdalcompare.py	2025-07-12 12:33:49.000000000 +0200
+++ gdal-3.11.3/swig/python/gdal-utils/osgeo_utils/gdalcompare.py	2025-08-21 17:51:03.198403926 +0200
@@ -1,4 +1,4 @@
-#!/usr/bin/env python3
+#!/usr/bin/python3
 # -*- coding: utf-8 -*-
 # ******************************************************************************
 #
diff -rupN gdal-3.11.3.old/swig/python/gdal-utils/osgeo_utils/gdal_edit.py gdal-3.11.3/swig/python/gdal-utils/osgeo_utils/gdal_edit.py
--- gdal-3.11.3.old/swig/python/gdal-utils/osgeo_utils/gdal_edit.py	2025-07-12 12:33:49.000000000 +0200
+++ gdal-3.11.3/swig/python/gdal-utils/osgeo_utils/gdal_edit.py	2025-08-21 17:49:28.501131853 +0200
@@ -1,4 +1,4 @@
-#!/usr/bin/env python3
+#!/usr/bin/python3
 # -*- coding: utf-8 -*-
 ###############################################################################
 #
diff -rupN gdal-3.11.3.old/swig/python/gdal-utils/osgeo_utils/gdal_fillnodata.py gdal-3.11.3/swig/python/gdal-utils/osgeo_utils/gdal_fillnodata.py
--- gdal-3.11.3.old/swig/python/gdal-utils/osgeo_utils/gdal_fillnodata.py	2025-07-12 12:33:49.000000000 +0200
+++ gdal-3.11.3/swig/python/gdal-utils/osgeo_utils/gdal_fillnodata.py	2025-08-21 17:49:36.760542070 +0200
@@ -1,4 +1,4 @@
-#!/usr/bin/env python3
+#!/usr/bin/python3
 # ******************************************************************************
 #
 #  Project:  GDAL Python Interface
diff -rupN gdal-3.11.3.old/swig/python/gdal-utils/osgeo_utils/gdal_merge.py gdal-3.11.3/swig/python/gdal-utils/osgeo_utils/gdal_merge.py
--- gdal-3.11.3.old/swig/python/gdal-utils/osgeo_utils/gdal_merge.py	2025-07-12 12:33:49.000000000 +0200
+++ gdal-3.11.3/swig/python/gdal-utils/osgeo_utils/gdal_merge.py	2025-08-21 17:49:47.785819278 +0200
@@ -1,4 +1,4 @@
-#!/usr/bin/env python3
+#!/usr/bin/python3
 ###############################################################################
 #
 # Project:  InSAR Peppers
diff -rupN gdal-3.11.3.old/swig/python/gdal-utils/osgeo_utils/gdalmove.py gdal-3.11.3/swig/python/gdal-utils/osgeo_utils/gdalmove.py
--- gdal-3.11.3.old/swig/python/gdal-utils/osgeo_utils/gdalmove.py	2025-07-12 12:33:49.000000000 +0200
+++ gdal-3.11.3/swig/python/gdal-utils/osgeo_utils/gdalmove.py	2025-08-21 17:51:10.667444856 +0200
@@ -1,4 +1,4 @@
-#!/usr/bin/env python3
+#!/usr/bin/python3
 # ******************************************************************************
 #
 #  Project:  GDAL Python Interface
diff -rupN gdal-3.11.3.old/swig/python/gdal-utils/osgeo_utils/gdal_pansharpen.py gdal-3.11.3/swig/python/gdal-utils/osgeo_utils/gdal_pansharpen.py
--- gdal-3.11.3.old/swig/python/gdal-utils/osgeo_utils/gdal_pansharpen.py	2025-07-12 12:33:49.000000000 +0200
+++ gdal-3.11.3/swig/python/gdal-utils/osgeo_utils/gdal_pansharpen.py	2025-08-21 17:49:55.459792949 +0200
@@ -1,4 +1,4 @@
-#!/usr/bin/env python3
+#!/usr/bin/python3
 # -*- coding: utf-8 -*-
 ###############################################################################
 #
diff -rupN gdal-3.11.3.old/swig/python/gdal-utils/osgeo_utils/gdal_polygonize.py gdal-3.11.3/swig/python/gdal-utils/osgeo_utils/gdal_polygonize.py
--- gdal-3.11.3.old/swig/python/gdal-utils/osgeo_utils/gdal_polygonize.py	2025-07-12 12:33:49.000000000 +0200
+++ gdal-3.11.3/swig/python/gdal-utils/osgeo_utils/gdal_polygonize.py	2025-08-21 17:50:05.071918222 +0200
@@ -1,4 +1,4 @@
-#!/usr/bin/env python3
+#!/usr/bin/python3
 # -*- coding: utf-8 -*-
 # ******************************************************************************
 #
diff -rupN gdal-3.11.3.old/swig/python/gdal-utils/osgeo_utils/gdal_proximity.py gdal-3.11.3/swig/python/gdal-utils/osgeo_utils/gdal_proximity.py
--- gdal-3.11.3.old/swig/python/gdal-utils/osgeo_utils/gdal_proximity.py	2025-07-12 12:33:49.000000000 +0200
+++ gdal-3.11.3/swig/python/gdal-utils/osgeo_utils/gdal_proximity.py	2025-08-21 17:50:13.199416712 +0200
@@ -1,4 +1,4 @@
-#!/usr/bin/env python3
+#!/usr/bin/python3
 # ******************************************************************************
 #
 #  Name:     gdalproximity
diff -rupN gdal-3.11.3.old/swig/python/gdal-utils/osgeo_utils/gdal_retile.py gdal-3.11.3/swig/python/gdal-utils/osgeo_utils/gdal_retile.py
--- gdal-3.11.3.old/swig/python/gdal-utils/osgeo_utils/gdal_retile.py	2025-07-12 12:33:49.000000000 +0200
+++ gdal-3.11.3/swig/python/gdal-utils/osgeo_utils/gdal_retile.py	2025-08-21 17:50:21.313676706 +0200
@@ -1,4 +1,4 @@
-#!/usr/bin/env python3
+#!/usr/bin/python3
 ###############################################################################
 #
 # Purpose:  Module for retiling (merging) tiles and building tiled pyramids
diff -rupN gdal-3.11.3.old/swig/python/gdal-utils/osgeo_utils/gdal_sieve.py gdal-3.11.3/swig/python/gdal-utils/osgeo_utils/gdal_sieve.py
--- gdal-3.11.3.old/swig/python/gdal-utils/osgeo_utils/gdal_sieve.py	2025-07-12 12:33:49.000000000 +0200
+++ gdal-3.11.3/swig/python/gdal-utils/osgeo_utils/gdal_sieve.py	2025-08-21 17:50:29.087673597 +0200
@@ -1,4 +1,4 @@
-#!/usr/bin/env python3
+#!/usr/bin/python3
 # ******************************************************************************
 #
 #  Project:  GDAL Python Interface
diff -rupN gdal-3.11.3.old/swig/python/gdal-utils/osgeo_utils/ogr_layer_algebra.py gdal-3.11.3/swig/python/gdal-utils/osgeo_utils/ogr_layer_algebra.py
--- gdal-3.11.3.old/swig/python/gdal-utils/osgeo_utils/ogr_layer_algebra.py	2025-07-12 12:33:49.000000000 +0200
+++ gdal-3.11.3/swig/python/gdal-utils/osgeo_utils/ogr_layer_algebra.py	2025-08-21 17:51:18.546408680 +0200
@@ -1,4 +1,4 @@
-#!/usr/bin/env python3
+#!/usr/bin/python3
 # -*- coding: utf-8 -*-
 # ******************************************************************************
 #
diff -rupN gdal-3.11.3.old/swig/python/gdal-utils/osgeo_utils/ogrmerge.py gdal-3.11.3/swig/python/gdal-utils/osgeo_utils/ogrmerge.py
--- gdal-3.11.3.old/swig/python/gdal-utils/osgeo_utils/ogrmerge.py	2025-07-12 12:33:49.000000000 +0200
+++ gdal-3.11.3/swig/python/gdal-utils/osgeo_utils/ogrmerge.py	2025-08-21 17:51:25.859102508 +0200
@@ -1,4 +1,4 @@
-#!/usr/bin/env python3
+#!/usr/bin/python3
 # -*- coding: utf-8 -*-
 ###############################################################################
 #
diff -rupN gdal-3.11.3.old/swig/python/gdal-utils/osgeo_utils/pct2rgb.py gdal-3.11.3/swig/python/gdal-utils/osgeo_utils/pct2rgb.py
--- gdal-3.11.3.old/swig/python/gdal-utils/osgeo_utils/pct2rgb.py	2025-07-12 12:33:49.000000000 +0200
+++ gdal-3.11.3/swig/python/gdal-utils/osgeo_utils/pct2rgb.py	2025-08-21 17:51:34.793947787 +0200
@@ -1,4 +1,4 @@
-#!/usr/bin/env python3
+#!/usr/bin/python3
 # ******************************************************************************
 #
 #  Name:     pct2rgb
diff -rupN gdal-3.11.3.old/swig/python/gdal-utils/osgeo_utils/rgb2pct.py gdal-3.11.3/swig/python/gdal-utils/osgeo_utils/rgb2pct.py
--- gdal-3.11.3.old/swig/python/gdal-utils/osgeo_utils/rgb2pct.py	2025-07-12 12:33:49.000000000 +0200
+++ gdal-3.11.3/swig/python/gdal-utils/osgeo_utils/rgb2pct.py	2025-08-21 17:49:10.651967972 +0200
@@ -1,4 +1,4 @@
-#!/usr/bin/env python3
+#!/usr/bin/python3
 # ******************************************************************************
 #
 #  Name:     rgb2pct
diff -rupN gdal-3.11.3.old/swig/python/gdal-utils/osgeo_utils/samples/assemblepoly.py gdal-3.11.3/swig/python/gdal-utils/osgeo_utils/samples/assemblepoly.py
--- gdal-3.11.3.old/swig/python/gdal-utils/osgeo_utils/samples/assemblepoly.py	2025-07-12 12:33:49.000000000 +0200
+++ gdal-3.11.3/swig/python/gdal-utils/osgeo_utils/samples/assemblepoly.py	2025-08-21 17:54:46.380672487 +0200
@@ -1,4 +1,4 @@
-#!/usr/bin/env python3
+#!/usr/bin/python3
 ###############################################################################
 #
 # Project:  OGR Python samples
diff -rupN gdal-3.11.3.old/swig/python/gdal-utils/osgeo_utils/samples/build_jp2_from_xml.py gdal-3.11.3/swig/python/gdal-utils/osgeo_utils/samples/build_jp2_from_xml.py
--- gdal-3.11.3.old/swig/python/gdal-utils/osgeo_utils/samples/build_jp2_from_xml.py	2025-07-12 12:33:49.000000000 +0200
+++ gdal-3.11.3/swig/python/gdal-utils/osgeo_utils/samples/build_jp2_from_xml.py	2025-08-21 17:54:54.224855412 +0200
@@ -1,4 +1,4 @@
-#!/usr/bin/env python3
+#!/usr/bin/python3
 # -*- coding: utf-8 -*-
 # ******************************************************************************
 #
diff -rupN gdal-3.11.3.old/swig/python/gdal-utils/osgeo_utils/samples/classify.py gdal-3.11.3/swig/python/gdal-utils/osgeo_utils/samples/classify.py
--- gdal-3.11.3.old/swig/python/gdal-utils/osgeo_utils/samples/classify.py	2025-07-12 12:33:49.000000000 +0200
+++ gdal-3.11.3/swig/python/gdal-utils/osgeo_utils/samples/classify.py	2025-08-21 17:55:00.842179472 +0200
@@ -1,4 +1,4 @@
-#!/usr/bin/env python3
+#!/usr/bin/python3
 # ******************************************************************************
 #
 #  Project:  GDAL
diff -rupN gdal-3.11.3.old/swig/python/gdal-utils/osgeo_utils/samples/crs2crs2grid.py gdal-3.11.3/swig/python/gdal-utils/osgeo_utils/samples/crs2crs2grid.py
--- gdal-3.11.3.old/swig/python/gdal-utils/osgeo_utils/samples/crs2crs2grid.py	2025-07-12 12:33:49.000000000 +0200
+++ gdal-3.11.3/swig/python/gdal-utils/osgeo_utils/samples/crs2crs2grid.py	2025-08-21 17:55:07.587830272 +0200
@@ -1,4 +1,4 @@
-#!/usr/bin/env python3
+#!/usr/bin/python3
 # ******************************************************************************
 #
 #  Project:  GDAL
diff -rupN gdal-3.11.3.old/swig/python/gdal-utils/osgeo_utils/samples/densify.py gdal-3.11.3/swig/python/gdal-utils/osgeo_utils/samples/densify.py
--- gdal-3.11.3.old/swig/python/gdal-utils/osgeo_utils/samples/densify.py	2025-07-12 12:33:49.000000000 +0200
+++ gdal-3.11.3/swig/python/gdal-utils/osgeo_utils/samples/densify.py	2025-08-21 17:55:17.491053457 +0200
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/python3
 # ******************************************************************************
 #
 #  Project:  GDAL
diff -rupN gdal-3.11.3.old/swig/python/gdal-utils/osgeo_utils/samples/dump_jp2.py gdal-3.11.3/swig/python/gdal-utils/osgeo_utils/samples/dump_jp2.py
--- gdal-3.11.3.old/swig/python/gdal-utils/osgeo_utils/samples/dump_jp2.py	2025-07-12 12:33:49.000000000 +0200
+++ gdal-3.11.3/swig/python/gdal-utils/osgeo_utils/samples/dump_jp2.py	2025-08-21 17:55:24.012886212 +0200
@@ -1,4 +1,4 @@
-#!/usr/bin/env python3
+#!/usr/bin/python3
 # -*- coding: utf-8 -*-
 # ******************************************************************************
 #
diff -rupN gdal-3.11.3.old/swig/python/gdal-utils/osgeo_utils/samples/epsg_tr.py gdal-3.11.3/swig/python/gdal-utils/osgeo_utils/samples/epsg_tr.py
--- gdal-3.11.3.old/swig/python/gdal-utils/osgeo_utils/samples/epsg_tr.py	2025-07-12 12:33:49.000000000 +0200
+++ gdal-3.11.3/swig/python/gdal-utils/osgeo_utils/samples/epsg_tr.py	2025-08-21 17:55:34.805030481 +0200
@@ -1,4 +1,4 @@
-#!/usr/bin/env python3
+#!/usr/bin/python3
 # -*- coding: utf-8 -*-
 # ******************************************************************************
 #
diff -rupN gdal-3.11.3.old/swig/python/gdal-utils/osgeo_utils/samples/esri2wkt.py gdal-3.11.3/swig/python/gdal-utils/osgeo_utils/samples/esri2wkt.py
--- gdal-3.11.3.old/swig/python/gdal-utils/osgeo_utils/samples/esri2wkt.py	2025-07-12 12:33:49.000000000 +0200
+++ gdal-3.11.3/swig/python/gdal-utils/osgeo_utils/samples/esri2wkt.py	2025-08-21 17:55:43.124261366 +0200
@@ -1,4 +1,4 @@
-#!/usr/bin/env python3
+#!/usr/bin/python3
 # ******************************************************************************
 #
 #  Project:  GDAL
diff -rupN gdal-3.11.3.old/swig/python/gdal-utils/osgeo_utils/samples/fft.py gdal-3.11.3/swig/python/gdal-utils/osgeo_utils/samples/fft.py
--- gdal-3.11.3.old/swig/python/gdal-utils/osgeo_utils/samples/fft.py	2025-07-12 12:33:49.000000000 +0200
+++ gdal-3.11.3/swig/python/gdal-utils/osgeo_utils/samples/fft.py	2025-08-21 17:55:50.394638565 +0200
@@ -1,4 +1,4 @@
-#!/usr/bin/env python3
+#!/usr/bin/python3
 ###############################################################################
 #
 # Project:  GDAL Python samples
diff -rupN gdal-3.11.3.old/swig/python/gdal-utils/osgeo_utils/samples/fix_gpkg.py gdal-3.11.3/swig/python/gdal-utils/osgeo_utils/samples/fix_gpkg.py
--- gdal-3.11.3.old/swig/python/gdal-utils/osgeo_utils/samples/fix_gpkg.py	2025-07-12 12:33:49.000000000 +0200
+++ gdal-3.11.3/swig/python/gdal-utils/osgeo_utils/samples/fix_gpkg.py	2025-08-21 17:55:57.634898557 +0200
@@ -1,4 +1,4 @@
-#!/usr/bin/env python3
+#!/usr/bin/python3
 # -*- coding: utf-8 -*-
 ###############################################################################
 #
diff -rupN gdal-3.11.3.old/swig/python/gdal-utils/osgeo_utils/samples/gcps2ogr.py gdal-3.11.3/swig/python/gdal-utils/osgeo_utils/samples/gcps2ogr.py
--- gdal-3.11.3.old/swig/python/gdal-utils/osgeo_utils/samples/gcps2ogr.py	2025-07-12 12:33:49.000000000 +0200
+++ gdal-3.11.3/swig/python/gdal-utils/osgeo_utils/samples/gcps2ogr.py	2025-08-21 17:56:04.351933366 +0200
@@ -1,4 +1,4 @@
-#!/usr/bin/env python3
+#!/usr/bin/python3
 # -*- coding: utf-8 -*-
 # ******************************************************************************
 #
diff -rupN gdal-3.11.3.old/swig/python/gdal-utils/osgeo_utils/samples/gcps2vec.py gdal-3.11.3/swig/python/gdal-utils/osgeo_utils/samples/gcps2vec.py
--- gdal-3.11.3.old/swig/python/gdal-utils/osgeo_utils/samples/gcps2vec.py	2025-07-12 12:33:49.000000000 +0200
+++ gdal-3.11.3/swig/python/gdal-utils/osgeo_utils/samples/gcps2vec.py	2025-08-21 17:56:10.617776356 +0200
@@ -1,4 +1,4 @@
-#!/usr/bin/env python3
+#!/usr/bin/python3
 # ******************************************************************************
 #
 #  Project:  GDAL
diff -rupN gdal-3.11.3.old/swig/python/gdal-utils/osgeo_utils/samples/gcps2wld.py gdal-3.11.3/swig/python/gdal-utils/osgeo_utils/samples/gcps2wld.py
--- gdal-3.11.3.old/swig/python/gdal-utils/osgeo_utils/samples/gcps2wld.py	2025-07-12 12:33:49.000000000 +0200
+++ gdal-3.11.3/swig/python/gdal-utils/osgeo_utils/samples/gcps2wld.py	2025-08-21 17:56:16.641291214 +0200
@@ -1,4 +1,4 @@
-#!/usr/bin/env python3
+#!/usr/bin/python3
 # ******************************************************************************
 #
 #  Name:     gcps2wld
diff -rupN gdal-3.11.3.old/swig/python/gdal-utils/osgeo_utils/samples/gdal2grd.py gdal-3.11.3/swig/python/gdal-utils/osgeo_utils/samples/gdal2grd.py
--- gdal-3.11.3.old/swig/python/gdal-utils/osgeo_utils/samples/gdal2grd.py	2025-07-12 12:33:49.000000000 +0200
+++ gdal-3.11.3/swig/python/gdal-utils/osgeo_utils/samples/gdal2grd.py	2025-08-21 17:57:55.647395726 +0200
@@ -1,4 +1,4 @@
-#!/usr/bin/env python3
+#!/usr/bin/python3
 ###############################################################################
 #
 # Project:  GDAL Python samples
diff -rupN gdal-3.11.3.old/swig/python/gdal-utils/osgeo_utils/samples/gdal_auth.py gdal-3.11.3/swig/python/gdal-utils/osgeo_utils/samples/gdal_auth.py
--- gdal-3.11.3.old/swig/python/gdal-utils/osgeo_utils/samples/gdal_auth.py	2025-07-12 12:33:49.000000000 +0200
+++ gdal-3.11.3/swig/python/gdal-utils/osgeo_utils/samples/gdal_auth.py	2025-08-21 17:56:23.658355195 +0200
@@ -1,4 +1,4 @@
-#!/usr/bin/env python3
+#!/usr/bin/python3
 # -*- coding: utf-8 -*-
 # ******************************************************************************
 #
diff -rupN gdal-3.11.3.old/swig/python/gdal-utils/osgeo_utils/samples/gdalbuildvrtofvrt.py gdal-3.11.3/swig/python/gdal-utils/osgeo_utils/samples/gdalbuildvrtofvrt.py
--- gdal-3.11.3.old/swig/python/gdal-utils/osgeo_utils/samples/gdalbuildvrtofvrt.py	2025-07-12 12:33:49.000000000 +0200
+++ gdal-3.11.3/swig/python/gdal-utils/osgeo_utils/samples/gdalbuildvrtofvrt.py	2025-08-21 17:58:03.809683556 +0200
@@ -1,4 +1,4 @@
-#!/usr/bin/env python3
+#!/usr/bin/python3
 # -*- coding: utf-8 -*-
 # ******************************************************************************
 #
diff -rupN gdal-3.11.3.old/swig/python/gdal-utils/osgeo_utils/samples/gdalchksum.py gdal-3.11.3/swig/python/gdal-utils/osgeo_utils/samples/gdalchksum.py
--- gdal-3.11.3.old/swig/python/gdal-utils/osgeo_utils/samples/gdalchksum.py	2025-07-12 12:33:49.000000000 +0200
+++ gdal-3.11.3/swig/python/gdal-utils/osgeo_utils/samples/gdalchksum.py	2025-08-21 17:58:11.261607743 +0200
@@ -1,4 +1,4 @@
-#!/usr/bin/env python3
+#!/usr/bin/python3
 # ******************************************************************************
 #
 #  Project:  GDAL
diff -rupN gdal-3.11.3.old/swig/python/gdal-utils/osgeo_utils/samples/gdalcopyproj.py gdal-3.11.3/swig/python/gdal-utils/osgeo_utils/samples/gdalcopyproj.py
--- gdal-3.11.3.old/swig/python/gdal-utils/osgeo_utils/samples/gdalcopyproj.py	2025-07-12 12:33:49.000000000 +0200
+++ gdal-3.11.3/swig/python/gdal-utils/osgeo_utils/samples/gdalcopyproj.py	2025-08-21 17:58:19.399803583 +0200
@@ -1,4 +1,4 @@
-#!/usr/bin/env python3
+#!/usr/bin/python3
 # ******************************************************************************
 #
 #  Name:     gdalcopyproj.py
diff -rupN gdal-3.11.3.old/swig/python/gdal-utils/osgeo_utils/samples/gdal_cp.py gdal-3.11.3/swig/python/gdal-utils/osgeo_utils/samples/gdal_cp.py
--- gdal-3.11.3.old/swig/python/gdal-utils/osgeo_utils/samples/gdal_cp.py	2025-07-12 12:33:49.000000000 +0200
+++ gdal-3.11.3/swig/python/gdal-utils/osgeo_utils/samples/gdal_cp.py	2025-08-21 17:56:30.801547023 +0200
@@ -1,4 +1,4 @@
-#!/usr/bin/env python3
+#!/usr/bin/python3
 ###############################################################################
 #
 #  Project:  GDAL samples
diff -rupN gdal-3.11.3.old/swig/python/gdal-utils/osgeo_utils/samples/gdal_create_pdf.py gdal-3.11.3/swig/python/gdal-utils/osgeo_utils/samples/gdal_create_pdf.py
--- gdal-3.11.3.old/swig/python/gdal-utils/osgeo_utils/samples/gdal_create_pdf.py	2025-07-12 12:33:49.000000000 +0200
+++ gdal-3.11.3/swig/python/gdal-utils/osgeo_utils/samples/gdal_create_pdf.py	2025-08-21 17:56:39.199711156 +0200
@@ -1,4 +1,4 @@
-#!/usr/bin/env python3
+#!/usr/bin/python3
 ###############################################################################
 #
 #  Project:  GDAL samples
diff -rupN gdal-3.11.3.old/swig/python/gdal-utils/osgeo_utils/samples/gdalfilter.py gdal-3.11.3/swig/python/gdal-utils/osgeo_utils/samples/gdalfilter.py
--- gdal-3.11.3.old/swig/python/gdal-utils/osgeo_utils/samples/gdalfilter.py	2025-07-12 12:33:49.000000000 +0200
+++ gdal-3.11.3/swig/python/gdal-utils/osgeo_utils/samples/gdalfilter.py	2025-08-21 17:58:28.570872904 +0200
@@ -1,4 +1,4 @@
-#!/usr/bin/env python3
+#!/usr/bin/python3
 ###############################################################################
 #
 # Project:  OGR Python samples
diff -rupN gdal-3.11.3.old/swig/python/gdal-utils/osgeo_utils/samples/gdalident.py gdal-3.11.3/swig/python/gdal-utils/osgeo_utils/samples/gdalident.py
--- gdal-3.11.3.old/swig/python/gdal-utils/osgeo_utils/samples/gdalident.py	2025-07-12 12:33:49.000000000 +0200
+++ gdal-3.11.3/swig/python/gdal-utils/osgeo_utils/samples/gdalident.py	2025-08-21 17:58:37.538279941 +0200
@@ -1,4 +1,4 @@
-#!/usr/bin/env python3
+#!/usr/bin/python3
 # ******************************************************************************
 #
 #  Project:  GDAL
diff -rupN gdal-3.11.3.old/swig/python/gdal-utils/osgeo_utils/samples/gdalimport.py gdal-3.11.3/swig/python/gdal-utils/osgeo_utils/samples/gdalimport.py
--- gdal-3.11.3.old/swig/python/gdal-utils/osgeo_utils/samples/gdalimport.py	2025-07-12 12:33:49.000000000 +0200
+++ gdal-3.11.3/swig/python/gdal-utils/osgeo_utils/samples/gdalimport.py	2025-08-21 17:58:51.321603934 +0200
@@ -1,4 +1,4 @@
-#!/usr/bin/env python3
+#!/usr/bin/python3
 # ******************************************************************************
 #
 #  Name:     gdalimport
diff -rupN gdal-3.11.3.old/swig/python/gdal-utils/osgeo_utils/samples/gdalinfo.py gdal-3.11.3/swig/python/gdal-utils/osgeo_utils/samples/gdalinfo.py
--- gdal-3.11.3.old/swig/python/gdal-utils/osgeo_utils/samples/gdalinfo.py	2025-07-12 12:33:49.000000000 +0200
+++ gdal-3.11.3/swig/python/gdal-utils/osgeo_utils/samples/gdalinfo.py	2025-08-21 17:59:01.740958948 +0200
@@ -1,4 +1,4 @@
-#!/usr/bin/env python3
+#!/usr/bin/python3
 # *****************************************************************************
 #
 # Project:  GDAL Utilities
diff -rupN gdal-3.11.3.old/swig/python/gdal-utils/osgeo_utils/samples/gdallocationinfo.py gdal-3.11.3/swig/python/gdal-utils/osgeo_utils/samples/gdallocationinfo.py
--- gdal-3.11.3.old/swig/python/gdal-utils/osgeo_utils/samples/gdallocationinfo.py	2025-07-12 12:33:49.000000000 +0200
+++ gdal-3.11.3/swig/python/gdal-utils/osgeo_utils/samples/gdallocationinfo.py	2025-08-21 17:59:11.178839200 +0200
@@ -1,4 +1,4 @@
-# !/usr/bin/env python3
+# !/usr/bin/python3
 ###############################################################################
 #
 # Project:  GDAL utils
diff -rupN gdal-3.11.3.old/swig/python/gdal-utils/osgeo_utils/samples/gdal_ls.py gdal-3.11.3/swig/python/gdal-utils/osgeo_utils/samples/gdal_ls.py
--- gdal-3.11.3.old/swig/python/gdal-utils/osgeo_utils/samples/gdal_ls.py	2025-07-12 12:33:49.000000000 +0200
+++ gdal-3.11.3/swig/python/gdal-utils/osgeo_utils/samples/gdal_ls.py	2025-08-21 17:56:46.522014957 +0200
@@ -1,4 +1,4 @@
-#!/usr/bin/env python3
+#!/usr/bin/python3
 # -*- coding: utf-8 -*-
 ###############################################################################
 #
diff -rupN gdal-3.11.3.old/swig/python/gdal-utils/osgeo_utils/samples/gdal_lut.py gdal-3.11.3/swig/python/gdal-utils/osgeo_utils/samples/gdal_lut.py
--- gdal-3.11.3.old/swig/python/gdal-utils/osgeo_utils/samples/gdal_lut.py	2025-07-12 12:33:49.000000000 +0200
+++ gdal-3.11.3/swig/python/gdal-utils/osgeo_utils/samples/gdal_lut.py	2025-08-21 17:56:54.553346496 +0200
@@ -1,4 +1,4 @@
-#!/usr/bin/env python3
+#!/usr/bin/python3
 # ******************************************************************************
 #  $Id: pct2rgb.py 13087 2007-11-26 20:56:29Z hobu $
 #
diff -rupN gdal-3.11.3.old/swig/python/gdal-utils/osgeo_utils/samples/gdal_minmax_location.py gdal-3.11.3/swig/python/gdal-utils/osgeo_utils/samples/gdal_minmax_location.py
--- gdal-3.11.3.old/swig/python/gdal-utils/osgeo_utils/samples/gdal_minmax_location.py	2025-07-12 12:33:49.000000000 +0200
+++ gdal-3.11.3/swig/python/gdal-utils/osgeo_utils/samples/gdal_minmax_location.py	2025-08-21 17:57:03.962826668 +0200
@@ -1,4 +1,4 @@
-# !/usr/bin/env python3
+# !/usr/bin/python3
 ###############################################################################
 # Project:  GDAL utils
 # Purpose:  Get min/max location
diff -rupN gdal-3.11.3.old/swig/python/gdal-utils/osgeo_utils/samples/gdal_mkdir.py gdal-3.11.3/swig/python/gdal-utils/osgeo_utils/samples/gdal_mkdir.py
--- gdal-3.11.3.old/swig/python/gdal-utils/osgeo_utils/samples/gdal_mkdir.py	2025-07-12 12:33:49.000000000 +0200
+++ gdal-3.11.3/swig/python/gdal-utils/osgeo_utils/samples/gdal_mkdir.py	2025-08-21 17:57:12.089950274 +0200
@@ -1,4 +1,4 @@
-#!/usr/bin/env python3
+#!/usr/bin/python3
 ###############################################################################
 #
 #  Project:  GDAL samples
diff -rupN gdal-3.11.3.old/swig/python/gdal-utils/osgeo_utils/samples/gdal_remove_towgs84.py gdal-3.11.3/swig/python/gdal-utils/osgeo_utils/samples/gdal_remove_towgs84.py
--- gdal-3.11.3.old/swig/python/gdal-utils/osgeo_utils/samples/gdal_remove_towgs84.py	2025-07-12 12:33:49.000000000 +0200
+++ gdal-3.11.3/swig/python/gdal-utils/osgeo_utils/samples/gdal_remove_towgs84.py	2025-08-21 17:57:22.506675375 +0200
@@ -1,4 +1,4 @@
-#!/usr/bin/env python3
+#!/usr/bin/python3
 # -*- coding: utf-8 -*-
 ###############################################################################
 #
diff -rupN gdal-3.11.3.old/swig/python/gdal-utils/osgeo_utils/samples/gdal_rmdir.py gdal-3.11.3/swig/python/gdal-utils/osgeo_utils/samples/gdal_rmdir.py
--- gdal-3.11.3.old/swig/python/gdal-utils/osgeo_utils/samples/gdal_rmdir.py	2025-07-12 12:33:49.000000000 +0200
+++ gdal-3.11.3/swig/python/gdal-utils/osgeo_utils/samples/gdal_rmdir.py	2025-08-21 17:57:40.372028815 +0200
@@ -1,4 +1,4 @@
-#!/usr/bin/env python3
+#!/usr/bin/python3
 ###############################################################################
 #
 #  Project:  GDAL samples
diff -rupN gdal-3.11.3.old/swig/python/gdal-utils/osgeo_utils/samples/gdal_rm.py gdal-3.11.3/swig/python/gdal-utils/osgeo_utils/samples/gdal_rm.py
--- gdal-3.11.3.old/swig/python/gdal-utils/osgeo_utils/samples/gdal_rm.py	2025-07-12 12:33:49.000000000 +0200
+++ gdal-3.11.3/swig/python/gdal-utils/osgeo_utils/samples/gdal_rm.py	2025-08-21 17:57:31.851940664 +0200
@@ -1,4 +1,4 @@
-#!/usr/bin/env python3
+#!/usr/bin/python3
 ###############################################################################
 #
 #  Project:  GDAL samples
diff -rupN gdal-3.11.3.old/swig/python/gdal-utils/osgeo_utils/samples/gdal_vrtmerge.py gdal-3.11.3/swig/python/gdal-utils/osgeo_utils/samples/gdal_vrtmerge.py
--- gdal-3.11.3.old/swig/python/gdal-utils/osgeo_utils/samples/gdal_vrtmerge.py	2025-07-12 12:33:49.000000000 +0200
+++ gdal-3.11.3/swig/python/gdal-utils/osgeo_utils/samples/gdal_vrtmerge.py	2025-08-21 17:57:48.007400717 +0200
@@ -1,4 +1,4 @@
-#!/usr/bin/env python3
+#!/usr/bin/python3
 # -*- coding: utf-8 -*-
 # Copyright (c) 2000, Atlantis Scientific Inc. (www.atlsci.com)
 # Copyright (C) 2005  Gabriel Ebner <ge@gabrielebner.at>
diff -rupN gdal-3.11.3.old/swig/python/gdal-utils/osgeo_utils/samples/get_soundg.py gdal-3.11.3/swig/python/gdal-utils/osgeo_utils/samples/get_soundg.py
--- gdal-3.11.3.old/swig/python/gdal-utils/osgeo_utils/samples/get_soundg.py	2025-07-12 12:33:49.000000000 +0200
+++ gdal-3.11.3/swig/python/gdal-utils/osgeo_utils/samples/get_soundg.py	2025-08-21 17:59:19.640277143 +0200
@@ -1,4 +1,4 @@
-#!/usr/bin/env python3
+#!/usr/bin/python3
 ###############################################################################
 #
 # Project:  OGR Python samples
diff -rupN gdal-3.11.3.old/swig/python/gdal-utils/osgeo_utils/samples/histrep.py gdal-3.11.3/swig/python/gdal-utils/osgeo_utils/samples/histrep.py
--- gdal-3.11.3.old/swig/python/gdal-utils/osgeo_utils/samples/histrep.py	2025-07-12 12:33:49.000000000 +0200
+++ gdal-3.11.3/swig/python/gdal-utils/osgeo_utils/samples/histrep.py	2025-08-21 17:59:29.452196187 +0200
@@ -1,4 +1,4 @@
-#!/usr/bin/env python3
+#!/usr/bin/python3
 ###############################################################################
 #
 # Project:  GDAL Python Samples
diff -rupN gdal-3.11.3.old/swig/python/gdal-utils/osgeo_utils/samples/hsv_merge.py gdal-3.11.3/swig/python/gdal-utils/osgeo_utils/samples/hsv_merge.py
--- gdal-3.11.3.old/swig/python/gdal-utils/osgeo_utils/samples/hsv_merge.py	2025-07-12 12:33:49.000000000 +0200
+++ gdal-3.11.3/swig/python/gdal-utils/osgeo_utils/samples/hsv_merge.py	2025-08-21 17:59:35.809636521 +0200
@@ -1,4 +1,4 @@
-#!/usr/bin/env python3
+#!/usr/bin/python3
 # ******************************************************************************
 #
 #  Project:  GDAL Python Interface
diff -rupN gdal-3.11.3.old/swig/python/gdal-utils/osgeo_utils/samples/jpeg_in_tiff_extract.py gdal-3.11.3/swig/python/gdal-utils/osgeo_utils/samples/jpeg_in_tiff_extract.py
--- gdal-3.11.3.old/swig/python/gdal-utils/osgeo_utils/samples/jpeg_in_tiff_extract.py	2025-07-12 12:33:49.000000000 +0200
+++ gdal-3.11.3/swig/python/gdal-utils/osgeo_utils/samples/jpeg_in_tiff_extract.py	2025-08-21 17:59:42.971768064 +0200
@@ -1,4 +1,4 @@
-#!/usr/bin/env python3
+#!/usr/bin/python3
 # -*- coding: utf-8 -*-
 ###############################################################################
 #
diff -rupN gdal-3.11.3.old/swig/python/gdal-utils/osgeo_utils/samples/load2odbc.py gdal-3.11.3/swig/python/gdal-utils/osgeo_utils/samples/load2odbc.py
--- gdal-3.11.3.old/swig/python/gdal-utils/osgeo_utils/samples/load2odbc.py	2025-07-12 12:33:49.000000000 +0200
+++ gdal-3.11.3/swig/python/gdal-utils/osgeo_utils/samples/load2odbc.py	2025-08-21 17:59:51.393966910 +0200
@@ -1,4 +1,4 @@
-#!/usr/bin/env python3
+#!/usr/bin/python3
 ###############################################################################
 #
 # Project:  OGR Python samples
diff -rupN gdal-3.11.3.old/swig/python/gdal-utils/osgeo_utils/samples/loslas2ntv2.py gdal-3.11.3/swig/python/gdal-utils/osgeo_utils/samples/loslas2ntv2.py
--- gdal-3.11.3.old/swig/python/gdal-utils/osgeo_utils/samples/loslas2ntv2.py	2025-07-12 12:33:49.000000000 +0200
+++ gdal-3.11.3/swig/python/gdal-utils/osgeo_utils/samples/loslas2ntv2.py	2025-08-21 17:59:58.814102313 +0200
@@ -1,4 +1,4 @@
-#!/usr/bin/env python3
+#!/usr/bin/python3
 # ******************************************************************************
 #  $Id: loslas2ntv2.py 19102 2010-03-15 23:41:44Z warmerdam $
 #
diff -rupN gdal-3.11.3.old/swig/python/gdal-utils/osgeo_utils/samples/magphase.py gdal-3.11.3/swig/python/gdal-utils/osgeo_utils/samples/magphase.py
--- gdal-3.11.3.old/swig/python/gdal-utils/osgeo_utils/samples/magphase.py	2025-07-12 12:33:49.000000000 +0200
+++ gdal-3.11.3/swig/python/gdal-utils/osgeo_utils/samples/magphase.py	2025-08-21 18:00:05.863075146 +0200
@@ -1,4 +1,4 @@
-#!/usr/bin/env python3
+#!/usr/bin/python3
 # ******************************************************************************
 #
 #  Project:  GDAL
diff -rupN gdal-3.11.3.old/swig/python/gdal-utils/osgeo_utils/samples/make_fuzzer_friendly_archive.py gdal-3.11.3/swig/python/gdal-utils/osgeo_utils/samples/make_fuzzer_friendly_archive.py
--- gdal-3.11.3.old/swig/python/gdal-utils/osgeo_utils/samples/make_fuzzer_friendly_archive.py	2025-07-12 12:33:49.000000000 +0200
+++ gdal-3.11.3/swig/python/gdal-utils/osgeo_utils/samples/make_fuzzer_friendly_archive.py	2025-08-21 18:00:15.898039437 +0200
@@ -1,4 +1,4 @@
-#!/usr/bin/env python3
+#!/usr/bin/python3
 # -*- coding: utf-8 -*-
 # ******************************************************************************
 #
diff -rupN gdal-3.11.3.old/swig/python/gdal-utils/osgeo_utils/samples/mkgraticule.py gdal-3.11.3/swig/python/gdal-utils/osgeo_utils/samples/mkgraticule.py
--- gdal-3.11.3.old/swig/python/gdal-utils/osgeo_utils/samples/mkgraticule.py	2025-07-12 12:33:49.000000000 +0200
+++ gdal-3.11.3/swig/python/gdal-utils/osgeo_utils/samples/mkgraticule.py	2025-08-21 18:00:25.205533867 +0200
@@ -1,4 +1,4 @@
-#!/usr/bin/env python3
+#!/usr/bin/python3
 ###############################################################################
 #
 # Project:  OGR Python samples
diff -rupN gdal-3.11.3.old/swig/python/gdal-utils/osgeo_utils/samples/ogr2vrt.py gdal-3.11.3/swig/python/gdal-utils/osgeo_utils/samples/ogr2vrt.py
--- gdal-3.11.3.old/swig/python/gdal-utils/osgeo_utils/samples/ogr2vrt.py	2025-07-12 12:33:49.000000000 +0200
+++ gdal-3.11.3/swig/python/gdal-utils/osgeo_utils/samples/ogr2vrt.py	2025-08-21 18:00:51.002415468 +0200
@@ -1,4 +1,4 @@
-#!/usr/bin/env python3
+#!/usr/bin/python3
 # -*- coding: utf-8 -*-
 ###############################################################################
 #
diff -rupN gdal-3.11.3.old/swig/python/gdal-utils/osgeo_utils/samples/ogr_build_junction_table.py gdal-3.11.3/swig/python/gdal-utils/osgeo_utils/samples/ogr_build_junction_table.py
--- gdal-3.11.3.old/swig/python/gdal-utils/osgeo_utils/samples/ogr_build_junction_table.py	2025-07-12 12:33:49.000000000 +0200
+++ gdal-3.11.3/swig/python/gdal-utils/osgeo_utils/samples/ogr_build_junction_table.py	2025-08-21 18:00:34.369798515 +0200
@@ -1,4 +1,4 @@
-#!/usr/bin/env python3
+#!/usr/bin/python3
 #
 # -*- coding: utf-8 -*-
 # ******************************************************************************
diff -rupN gdal-3.11.3.old/swig/python/gdal-utils/osgeo_utils/samples/ogr_dispatch.py gdal-3.11.3/swig/python/gdal-utils/osgeo_utils/samples/ogr_dispatch.py
--- gdal-3.11.3.old/swig/python/gdal-utils/osgeo_utils/samples/ogr_dispatch.py	2025-07-12 12:33:49.000000000 +0200
+++ gdal-3.11.3/swig/python/gdal-utils/osgeo_utils/samples/ogr_dispatch.py	2025-08-21 18:00:41.699090079 +0200
@@ -1,4 +1,4 @@
-#!/usr/bin/env python3
+#!/usr/bin/python3
 # -*- coding: utf-8 -*-
 ###############################################################################
 #
diff -rupN gdal-3.11.3.old/swig/python/gdal-utils/osgeo_utils/samples/ogrinfo.py gdal-3.11.3/swig/python/gdal-utils/osgeo_utils/samples/ogrinfo.py
--- gdal-3.11.3.old/swig/python/gdal-utils/osgeo_utils/samples/ogrinfo.py	2025-07-12 12:33:49.000000000 +0200
+++ gdal-3.11.3/swig/python/gdal-utils/osgeo_utils/samples/ogrinfo.py	2025-08-21 18:00:58.487391023 +0200
@@ -1,4 +1,4 @@
-#!/usr/bin/env python3
+#!/usr/bin/python3
 # *****************************************************************************
 #
 # Project:  OpenGIS Simple Features Reference Implementation
diff -rupN gdal-3.11.3.old/swig/python/gdal-utils/osgeo_utils/samples/ogrupdate.py gdal-3.11.3/swig/python/gdal-utils/osgeo_utils/samples/ogrupdate.py
--- gdal-3.11.3.old/swig/python/gdal-utils/osgeo_utils/samples/ogrupdate.py	2025-07-12 12:33:49.000000000 +0200
+++ gdal-3.11.3/swig/python/gdal-utils/osgeo_utils/samples/ogrupdate.py	2025-08-21 18:01:07.611433822 +0200
@@ -1,4 +1,4 @@
-#!/usr/bin/env python3
+#!/usr/bin/python3
 # -*- coding: utf-8 -*-
 ###############################################################################
 #
diff -rupN gdal-3.11.3.old/swig/python/gdal-utils/osgeo_utils/samples/rel.py gdal-3.11.3/swig/python/gdal-utils/osgeo_utils/samples/rel.py
--- gdal-3.11.3.old/swig/python/gdal-utils/osgeo_utils/samples/rel.py	2025-07-12 12:33:49.000000000 +0200
+++ gdal-3.11.3/swig/python/gdal-utils/osgeo_utils/samples/rel.py	2025-08-21 18:01:14.274302239 +0200
@@ -1,4 +1,4 @@
-#!/usr/bin/env python3
+#!/usr/bin/python3
 ###############################################################################
 #
 # Project:  GDAL Python samples
diff -rupN gdal-3.11.3.old/swig/python/gdal-utils/osgeo_utils/samples/tigerpoly.py gdal-3.11.3/swig/python/gdal-utils/osgeo_utils/samples/tigerpoly.py
--- gdal-3.11.3.old/swig/python/gdal-utils/osgeo_utils/samples/tigerpoly.py	2025-07-12 12:33:49.000000000 +0200
+++ gdal-3.11.3/swig/python/gdal-utils/osgeo_utils/samples/tigerpoly.py	2025-08-21 18:01:22.263520945 +0200
@@ -1,4 +1,4 @@
-#!/usr/bin/env python3
+#!/usr/bin/python3
 ###############################################################################
 #
 # Project:  OGR Python samples
diff -rupN gdal-3.11.3.old/swig/python/gdal-utils/osgeo_utils/samples/tile_extent_from_raster.py gdal-3.11.3/swig/python/gdal-utils/osgeo_utils/samples/tile_extent_from_raster.py
--- gdal-3.11.3.old/swig/python/gdal-utils/osgeo_utils/samples/tile_extent_from_raster.py	2025-07-12 12:33:49.000000000 +0200
+++ gdal-3.11.3/swig/python/gdal-utils/osgeo_utils/samples/tile_extent_from_raster.py	2025-08-21 18:01:29.411940905 +0200
@@ -1,4 +1,4 @@
-#!/usr/bin/env python3
+#!/usr/bin/python3
 # -*- coding: utf-8 -*-
 # ******************************************************************************
 #
diff -rupN gdal-3.11.3.old/swig/python/gdal-utils/osgeo_utils/samples/tolatlong.py gdal-3.11.3/swig/python/gdal-utils/osgeo_utils/samples/tolatlong.py
--- gdal-3.11.3.old/swig/python/gdal-utils/osgeo_utils/samples/tolatlong.py	2025-07-12 12:33:49.000000000 +0200
+++ gdal-3.11.3/swig/python/gdal-utils/osgeo_utils/samples/tolatlong.py	2025-08-21 18:01:38.101899558 +0200
@@ -1,4 +1,4 @@
-#!/usr/bin/env python3
+#!/usr/bin/python3
 ###############################################################################
 #
 # Project:  GDAL Python samples
diff -rupN gdal-3.11.3.old/swig/python/gdal-utils/osgeo_utils/samples/validate_cloud_optimized_geotiff.py gdal-3.11.3/swig/python/gdal-utils/osgeo_utils/samples/validate_cloud_optimized_geotiff.py
--- gdal-3.11.3.old/swig/python/gdal-utils/osgeo_utils/samples/validate_cloud_optimized_geotiff.py	2025-07-12 12:33:49.000000000 +0200
+++ gdal-3.11.3/swig/python/gdal-utils/osgeo_utils/samples/validate_cloud_optimized_geotiff.py	2025-08-21 18:01:55.466767766 +0200
@@ -1,4 +1,4 @@
-#!/usr/bin/env python3
+#!/usr/bin/python3
 # -*- coding: utf-8 -*-
 # *****************************************************************************
 #
diff -rupN gdal-3.11.3.old/swig/python/gdal-utils/osgeo_utils/samples/validate_geoparquet.py gdal-3.11.3/swig/python/gdal-utils/osgeo_utils/samples/validate_geoparquet.py
--- gdal-3.11.3.old/swig/python/gdal-utils/osgeo_utils/samples/validate_geoparquet.py	2025-07-12 12:33:49.000000000 +0200
+++ gdal-3.11.3/swig/python/gdal-utils/osgeo_utils/samples/validate_geoparquet.py	2025-08-21 18:02:03.912063830 +0200
@@ -1,4 +1,4 @@
-#!/usr/bin/env python3
+#!/usr/bin/python3
 # -*- coding: utf-8 -*-
 ###############################################################################
 #
diff -rupN gdal-3.11.3.old/swig/python/gdal-utils/osgeo_utils/samples/validate_gpkg.py gdal-3.11.3/swig/python/gdal-utils/osgeo_utils/samples/validate_gpkg.py
--- gdal-3.11.3.old/swig/python/gdal-utils/osgeo_utils/samples/validate_gpkg.py	2025-07-12 12:33:49.000000000 +0200
+++ gdal-3.11.3/swig/python/gdal-utils/osgeo_utils/samples/validate_gpkg.py	2025-08-21 18:02:13.704579333 +0200
@@ -1,4 +1,4 @@
-#!/usr/bin/env python3
+#!/usr/bin/python3
 # -*- coding: utf-8 -*-
 ###############################################################################
 #
diff -rupN gdal-3.11.3.old/swig/python/gdal-utils/osgeo_utils/samples/validate_jp2.py gdal-3.11.3/swig/python/gdal-utils/osgeo_utils/samples/validate_jp2.py
--- gdal-3.11.3.old/swig/python/gdal-utils/osgeo_utils/samples/validate_jp2.py	2025-07-12 12:33:49.000000000 +0200
+++ gdal-3.11.3/swig/python/gdal-utils/osgeo_utils/samples/validate_jp2.py	2025-08-21 18:02:21.611408570 +0200
@@ -1,4 +1,4 @@
-#!/usr/bin/env python3
+#!/usr/bin/python3
 # -*- coding: utf-8 -*-
 # ******************************************************************************
 #
diff -rupN gdal-3.11.3.old/swig/python/gdal-utils/osgeo_utils/samples/val_repl.py gdal-3.11.3/swig/python/gdal-utils/osgeo_utils/samples/val_repl.py
--- gdal-3.11.3.old/swig/python/gdal-utils/osgeo_utils/samples/val_repl.py	2025-07-12 12:33:49.000000000 +0200
+++ gdal-3.11.3/swig/python/gdal-utils/osgeo_utils/samples/val_repl.py	2025-08-21 18:01:45.293886082 +0200
@@ -1,4 +1,4 @@
-#!/usr/bin/env python3
+#!/usr/bin/python3
 ###############################################################################
 #
 # Project:  GDAL Python samples
diff -rupN gdal-3.11.3.old/swig/python/gdal-utils/osgeo_utils/samples/vec_tr.py gdal-3.11.3/swig/python/gdal-utils/osgeo_utils/samples/vec_tr.py
--- gdal-3.11.3.old/swig/python/gdal-utils/osgeo_utils/samples/vec_tr.py	2025-07-12 12:33:49.000000000 +0200
+++ gdal-3.11.3/swig/python/gdal-utils/osgeo_utils/samples/vec_tr.py	2025-08-21 18:02:28.898737979 +0200
@@ -1,4 +1,4 @@
-#!/usr/bin/env python3
+#!/usr/bin/python3
 ###############################################################################
 #
 # Project:  OGR Python samples
diff -rupN gdal-3.11.3.old/swig/python/gdal-utils/osgeo_utils/samples/vec_tr_spat.py gdal-3.11.3/swig/python/gdal-utils/osgeo_utils/samples/vec_tr_spat.py
--- gdal-3.11.3.old/swig/python/gdal-utils/osgeo_utils/samples/vec_tr_spat.py	2025-07-12 12:33:49.000000000 +0200
+++ gdal-3.11.3/swig/python/gdal-utils/osgeo_utils/samples/vec_tr_spat.py	2025-08-21 18:02:35.871388027 +0200
@@ -1,4 +1,4 @@
-#!/usr/bin/env python3
+#!/usr/bin/python3
 ###############################################################################
 #
 # Project:  OGR Python samples
diff -rupN gdal-3.11.3.old/swig/python/gdal-utils/osgeo_utils/samples/wcs_virtds_params.py gdal-3.11.3/swig/python/gdal-utils/osgeo_utils/samples/wcs_virtds_params.py
--- gdal-3.11.3.old/swig/python/gdal-utils/osgeo_utils/samples/wcs_virtds_params.py	2025-07-12 12:33:49.000000000 +0200
+++ gdal-3.11.3/swig/python/gdal-utils/osgeo_utils/samples/wcs_virtds_params.py	2025-08-21 17:54:35.564968576 +0200
@@ -1,4 +1,4 @@
-#!/usr/bin/env python3
+#!/usr/bin/python3
 #
 # -*- coding: utf-8 -*-
 # ******************************************************************************
diff -rupN gdal-3.11.3.old/swig/python/gdal-utils/scripts/gdal2tiles.py gdal-3.11.3/swig/python/gdal-utils/scripts/gdal2tiles.py
--- gdal-3.11.3.old/swig/python/gdal-utils/scripts/gdal2tiles.py	2025-07-12 12:33:49.000000000 +0200
+++ gdal-3.11.3/swig/python/gdal-utils/scripts/gdal2tiles.py	2025-08-21 18:22:28.330403048 +0200
@@ -1,4 +1,4 @@
-#!/usr/bin/env python3
+#!/usr/bin/python3
 
 import sys
 
diff -rupN gdal-3.11.3.old/swig/python/gdal-utils/scripts/gdal2xyz.py gdal-3.11.3/swig/python/gdal-utils/scripts/gdal2xyz.py
--- gdal-3.11.3.old/swig/python/gdal-utils/scripts/gdal2xyz.py	2025-07-12 12:33:49.000000000 +0200
+++ gdal-3.11.3/swig/python/gdal-utils/scripts/gdal2xyz.py	2025-08-21 18:22:36.768494136 +0200
@@ -1,4 +1,4 @@
-#!/usr/bin/env python3
+#!/usr/bin/python3
 
 import sys
 
diff -rupN gdal-3.11.3.old/swig/python/gdal-utils/scripts/gdalattachpct.py gdal-3.11.3/swig/python/gdal-utils/scripts/gdalattachpct.py
--- gdal-3.11.3.old/swig/python/gdal-utils/scripts/gdalattachpct.py	2025-07-12 12:33:49.000000000 +0200
+++ gdal-3.11.3/swig/python/gdal-utils/scripts/gdalattachpct.py	2025-08-21 18:22:43.210355601 +0200
@@ -1,4 +1,4 @@
-#!/usr/bin/env python3
+#!/usr/bin/python3
 
 import sys
 
diff -rupN gdal-3.11.3.old/swig/python/gdal-utils/scripts/gdal_calc.py gdal-3.11.3/swig/python/gdal-utils/scripts/gdal_calc.py
--- gdal-3.11.3.old/swig/python/gdal-utils/scripts/gdal_calc.py	2025-07-12 12:33:49.000000000 +0200
+++ gdal-3.11.3/swig/python/gdal-utils/scripts/gdal_calc.py	2025-08-21 18:21:24.940856161 +0200
@@ -1,4 +1,4 @@
-#!/usr/bin/env python3
+#!/usr/bin/python3
 
 import sys
 
diff -rupN gdal-3.11.3.old/swig/python/gdal-utils/scripts/gdalcompare.py gdal-3.11.3/swig/python/gdal-utils/scripts/gdalcompare.py
--- gdal-3.11.3.old/swig/python/gdal-utils/scripts/gdalcompare.py	2025-07-12 12:33:49.000000000 +0200
+++ gdal-3.11.3/swig/python/gdal-utils/scripts/gdalcompare.py	2025-08-21 18:22:51.180755926 +0200
@@ -1,4 +1,4 @@
-#!/usr/bin/env python3
+#!/usr/bin/python3
 
 import sys
 
diff -rupN gdal-3.11.3.old/swig/python/gdal-utils/scripts/gdal_edit.py gdal-3.11.3/swig/python/gdal-utils/scripts/gdal_edit.py
--- gdal-3.11.3.old/swig/python/gdal-utils/scripts/gdal_edit.py	2025-07-12 12:33:49.000000000 +0200
+++ gdal-3.11.3/swig/python/gdal-utils/scripts/gdal_edit.py	2025-08-21 18:21:31.742902807 +0200
@@ -1,4 +1,4 @@
-#!/usr/bin/env python3
+#!/usr/bin/python3
 
 import sys
 
diff -rupN gdal-3.11.3.old/swig/python/gdal-utils/scripts/gdal_fillnodata.py gdal-3.11.3/swig/python/gdal-utils/scripts/gdal_fillnodata.py
--- gdal-3.11.3.old/swig/python/gdal-utils/scripts/gdal_fillnodata.py	2025-07-12 12:33:49.000000000 +0200
+++ gdal-3.11.3/swig/python/gdal-utils/scripts/gdal_fillnodata.py	2025-08-21 18:21:38.669240797 +0200
@@ -1,4 +1,4 @@
-#!/usr/bin/env python3
+#!/usr/bin/python3
 
 import sys
 
diff -rupN gdal-3.11.3.old/swig/python/gdal-utils/scripts/gdal_merge.py gdal-3.11.3/swig/python/gdal-utils/scripts/gdal_merge.py
--- gdal-3.11.3.old/swig/python/gdal-utils/scripts/gdal_merge.py	2025-07-12 12:33:49.000000000 +0200
+++ gdal-3.11.3/swig/python/gdal-utils/scripts/gdal_merge.py	2025-08-21 18:21:47.327043917 +0200
@@ -1,4 +1,4 @@
-#!/usr/bin/env python3
+#!/usr/bin/python3
 
 import sys
 
diff -rupN gdal-3.11.3.old/swig/python/gdal-utils/scripts/gdalmove.py gdal-3.11.3/swig/python/gdal-utils/scripts/gdalmove.py
--- gdal-3.11.3.old/swig/python/gdal-utils/scripts/gdalmove.py	2025-07-12 12:33:49.000000000 +0200
+++ gdal-3.11.3/swig/python/gdal-utils/scripts/gdalmove.py	2025-08-21 18:22:57.443430123 +0200
@@ -1,4 +1,4 @@
-#!/usr/bin/env python3
+#!/usr/bin/python3
 
 import sys
 
diff -rupN gdal-3.11.3.old/swig/python/gdal-utils/scripts/gdal_pansharpen.py gdal-3.11.3/swig/python/gdal-utils/scripts/gdal_pansharpen.py
--- gdal-3.11.3.old/swig/python/gdal-utils/scripts/gdal_pansharpen.py	2025-07-12 12:33:49.000000000 +0200
+++ gdal-3.11.3/swig/python/gdal-utils/scripts/gdal_pansharpen.py	2025-08-21 18:21:54.989619557 +0200
@@ -1,4 +1,4 @@
-#!/usr/bin/env python3
+#!/usr/bin/python3
 
 import sys
 
diff -rupN gdal-3.11.3.old/swig/python/gdal-utils/scripts/gdal_polygonize.py gdal-3.11.3/swig/python/gdal-utils/scripts/gdal_polygonize.py
--- gdal-3.11.3.old/swig/python/gdal-utils/scripts/gdal_polygonize.py	2025-07-12 12:33:49.000000000 +0200
+++ gdal-3.11.3/swig/python/gdal-utils/scripts/gdal_polygonize.py	2025-08-21 18:22:01.319337259 +0200
@@ -1,4 +1,4 @@
-#!/usr/bin/env python3
+#!/usr/bin/python3
 
 import sys
 
diff -rupN gdal-3.11.3.old/swig/python/gdal-utils/scripts/gdal_proximity.py gdal-3.11.3/swig/python/gdal-utils/scripts/gdal_proximity.py
--- gdal-3.11.3.old/swig/python/gdal-utils/scripts/gdal_proximity.py	2025-07-12 12:33:49.000000000 +0200
+++ gdal-3.11.3/swig/python/gdal-utils/scripts/gdal_proximity.py	2025-08-21 18:22:07.788102189 +0200
@@ -1,4 +1,4 @@
-#!/usr/bin/env python3
+#!/usr/bin/python3
 
 import sys
 
diff -rupN gdal-3.11.3.old/swig/python/gdal-utils/scripts/gdal_retile.py gdal-3.11.3/swig/python/gdal-utils/scripts/gdal_retile.py
--- gdal-3.11.3.old/swig/python/gdal-utils/scripts/gdal_retile.py	2025-07-12 12:33:49.000000000 +0200
+++ gdal-3.11.3/swig/python/gdal-utils/scripts/gdal_retile.py	2025-08-21 18:22:16.383937471 +0200
@@ -1,4 +1,4 @@
-#!/usr/bin/env python3
+#!/usr/bin/python3
 
 import sys
 
diff -rupN gdal-3.11.3.old/swig/python/gdal-utils/scripts/gdal_sieve.py gdal-3.11.3/swig/python/gdal-utils/scripts/gdal_sieve.py
--- gdal-3.11.3.old/swig/python/gdal-utils/scripts/gdal_sieve.py	2025-07-12 12:33:49.000000000 +0200
+++ gdal-3.11.3/swig/python/gdal-utils/scripts/gdal_sieve.py	2025-08-21 18:22:22.247546299 +0200
@@ -1,4 +1,4 @@
-#!/usr/bin/env python3
+#!/usr/bin/python3
 
 import sys
 
diff -rupN gdal-3.11.3.old/swig/python/gdal-utils/scripts/ogr_layer_algebra.py gdal-3.11.3/swig/python/gdal-utils/scripts/ogr_layer_algebra.py
--- gdal-3.11.3.old/swig/python/gdal-utils/scripts/ogr_layer_algebra.py	2025-07-12 12:33:49.000000000 +0200
+++ gdal-3.11.3/swig/python/gdal-utils/scripts/ogr_layer_algebra.py	2025-08-21 18:23:03.313801176 +0200
@@ -1,4 +1,4 @@
-#!/usr/bin/env python3
+#!/usr/bin/python3
 
 import sys
 
diff -rupN gdal-3.11.3.old/swig/python/gdal-utils/scripts/ogrmerge.py gdal-3.11.3/swig/python/gdal-utils/scripts/ogrmerge.py
--- gdal-3.11.3.old/swig/python/gdal-utils/scripts/ogrmerge.py	2025-07-12 12:33:49.000000000 +0200
+++ gdal-3.11.3/swig/python/gdal-utils/scripts/ogrmerge.py	2025-08-21 18:23:09.258312502 +0200
@@ -1,4 +1,4 @@
-#!/usr/bin/env python3
+#!/usr/bin/python3
 
 import sys
 
diff -rupN gdal-3.11.3.old/swig/python/gdal-utils/scripts/pct2rgb.py gdal-3.11.3/swig/python/gdal-utils/scripts/pct2rgb.py
--- gdal-3.11.3.old/swig/python/gdal-utils/scripts/pct2rgb.py	2025-07-12 12:33:49.000000000 +0200
+++ gdal-3.11.3/swig/python/gdal-utils/scripts/pct2rgb.py	2025-08-21 18:23:16.008818102 +0200
@@ -1,4 +1,4 @@
-#!/usr/bin/env python3
+#!/usr/bin/python3
 
 import sys
 
diff -rupN gdal-3.11.3.old/swig/python/gdal-utils/scripts/rgb2pct.py gdal-3.11.3/swig/python/gdal-utils/scripts/rgb2pct.py
--- gdal-3.11.3.old/swig/python/gdal-utils/scripts/rgb2pct.py	2025-07-12 12:33:49.000000000 +0200
+++ gdal-3.11.3/swig/python/gdal-utils/scripts/rgb2pct.py	2025-08-21 18:21:16.385025418 +0200
@@ -1,4 +1,4 @@
-#!/usr/bin/env python3
+#!/usr/bin/python3
 
 import sys
 
