--- gdal-2.0.1/swig/python/GNUmakefile.orig	2016-01-18 20:36:48.569943935 +0000
+++ gdal-2.0.1/swig/python/GNUmakefile	2016-01-18 20:37:03.926751948 +0000
@@ -60,7 +60,7 @@
 	-grep "1\.3\.40" extensions/gdal_wrap.cpp >/dev/null && cat python3.2.patch | sed "s/gdal_wrap/gdal_array_wrap/" | patch -p0
     
 build:
-	$(PYTHON) setup.py build
+	CC=clang CXX=clang++ $(PYTHON) setup.py build
 
 egg:
 	$(PYTHON) setup.py bdist_egg 
